script/options.js

```diff
@@ -18,6 +18,7 @@ const COMMAND_LINE_OPTIONS_DEFINITIONS = [
   { name: 'protocol', type: String, defaultValue: 'http' },
   { name: 'public', type: String, defaultValue: null },
   { name: 'destination', type: String, defaultValue: null },
+  { name: 'content-deployment', type: Boolean, defaultValue: false },
```

Comment:
This is the command flag that is used to determine whether to download or compile Webpack assets.